lib/dm-is-reflective/adapters/postgres_adapter.rb

```diff
@@ -5,7 +5,7 @@ module DmIsReflective::PostgresAdapter
   def storages
     sql = <<-SQL
       SELECT table_name FROM "information_schema"."tables"
-      WHERE table_schema = current_schema()
+      WHERE table_schema = current_schema() AND table_type LIKE 'BASE TABLE'
```

Comment:
I think you should use `table_type = 'BASE TABLE'`. There's no `%` in the string, so we don't need to use `LIKE` and we could just match exactly.

Reply:
I also thought the same but, after trying it I realized that Porstgres was not accepting it.
Now I tried with pgAdmin client and it seems to work. However, if I change it in the ruby code, it does not work.

To be honest, I am no SQL expert to understand the performance, concerns of the difference, or why it decides to fail with the '='. Maybe it is some sort of character escaping issue. 

Looking forward for your comment.

Reply:
Umm... I know that you already tried it, but could you please try again? Or maybe query the value so that we could take a look? What's the result of:

``` sql
SELECT table_name, table_type FROM "information_schema"."tables"
WHERE table_schema = current_schema();
```

?

To me, it's less about the performance (it should actually be the same), it's more about it doesn't make sense. I would hesitate to consider a solution we don't really understand a solution. For a like pattern, only `%` and `_` should matter. There are none of it.

Reply:
@philfine Did you see my above comment?

Reply:
In my database the result is:
```
SELECT table_name, table_type FROM "information_schema"."tables"
WHERE table_schema = current_schema(); 

"geography_columns";"VIEW"
"geometry_columns";"VIEW"
"spatial_ref_sys";"BASE TABLE"
"raster_columns";"VIEW"
"raster_overviews";"VIEW"
"a";"BASE TABLE"
"b";"BASE TABLE"
"c";"BASE TABLE"
```

Strange enough, I changed the LIKE to '=' and now it seems to work correctly.
I don't really know how this all happened.

For me it is perfect to be '=' instead of LIKE. Changing in the patch.

Cheers,
Cupertino